extension/content.js

```diff
@@ -232,6 +284,12 @@ $(document).on('keydown', event => {
 		indentInput(event.target);
 		return false;
 	}
+
+	if ((event.which === 38 || event.which === 40) && event.shiftKey && event.metaKey) {
+		event.preventDefault();
+		lineUpInput(event.target, event.which === 40);
+		return false;
+	}
```

Comment:
Can you move all the logic into a separate file?

All that needs to be here is just the check for whether to activate or not:

```js
if (lineUpInput(event) {
  return false;
}
```